src/main/java/org/apache/maven/shared/scriptinterpreter/GroovyScriptInterpreter.java

```diff
@@ -67,16 +65,8 @@ public Object evaluateScript(
                     new Binding(globalVariables),
                     new CompilerConfiguration(CompilerConfiguration.DEFAULT));
 
-            try {
-                return interpreter.evaluate(script);
-            } catch (ThreadDeath e) {
-                throw e;
-            } catch (PowerAssertionError e) {
-                throw new ScriptEvaluationException("Assertion Error", e);
```

Comment:
I added it in https://github.com/apache/maven-script-interpreter/pull/14/files
Now I don't remember exactly why 😄 

please check how assertions error will be reported by invoker

Reply:
only 1 IT https://github.com/apache/maven-invoker-plugin/blob/d8a48716b2376a977ccdb0470f146dc5c1211258/src/it/script-streamLogs-false/verify.groovy#L45

Reply:
ok, code looks simpler